### PR TITLE
Small catalog fixes 

### PR DESCRIPTION
**Pull request by nicolejkeeney**
This PR addresses some small comments from reviewers on the main catalog restructure PR. 

1. Rename `catalog_utils.py` to `catalog_convert.py` 
2. Add a space between the resolution options; i.e. "45km" --> "45 km" 
3. Rename the functions within `catalog_convert.py` to be more descriptive 
4. Add back in an error statement if the user has selected "Append Historical" but no SSP 
5. Rename `selections.dataset` to `selections.downscaling_method`
6. Avoid creating large dask chunks and silence warning. This is done by setting the following config at the top of `data_loaders.py`:  ```dask.config.set({"array.slicing.split_large_chunks": True})```
7. Fix a small bug in `app.view`

### CODE REVIEW

**Review by vicford — APPROVED** [review]
Looks good to me

**Review by elehmer — APPROVED** [review]
LGTM! Seems to retrieve data just fine.

**Review by elehmer — COMMENTED** [review]
Noticed the tests failing. See comment...

File: setup.cfg [hunk 1]
```diff
@@ -45,4 +45,6 @@ docs =
     sphinx-book-theme
 
 [options.package_data]
-* = data/*.csv
+* =
+    data/cmaps/*.txt
```
[the remark this answers is not in the document]
Comment (nicolejkeeney) [reply setup.cfg:49]: Thanks Eric, couldn't figure out why they were failing. I'll try your suggestion.

[the remark this answers is not in the document]
Comment (elehmer) [reply setup.cfg]: Well, that didn't work :/...Back to the drawing board...

**Pull request merged**